code/game/objects/items/food/soup.dm

```diff
@@ -81,7 +81,7 @@
 	name = "nettle soup"
 	desc = "To think, the botanist would've beat you to death with one of these."
 	icon_state = "nettlesoup"
-	food_reagents = list(/datum/reagent/consumable/nutriment = 5, /datum/reagent/consumable/nutriment/protein = 2, /datum/reagent/water = 5, /datum/reagent/consumable/nutriment/vitamin = 9, /datum/reagent/medicine/omnizine = 5)
```

Comment:
Nettle Soup doesn't inherit omnizine from any of its components, it seems to be a joke/reference to nettle soup being healthy irl. 

You can still grind up nettle soup and get the omnizine removed. It's a bit inelegant as a solution but it is a kind of small annoyance that you can deal with via future tech (chem master), which is the kind of wrench in the system that quirks are meant to create.

The other cases here seem to be redundant inheritance, but this is one case where the ingredients come together to make something that's more than the sum of their parts, which is neat.

Reply:
If I'm gonna be completely honest: I didn't even know nettles were a real plant, let alone soup ingredient.

If you think it's better to leave in the omnizine, I'll make that change, but I don't know if "healthy irl" is exactly "5u of literal medicine". Granted, "carrots good for the eyes" translated into oculine, so.... Anyhow. I think the high vitamin content already covers being "healthy."

Reply:
Your PR body details that the point of doing this was so that people could intentionally grow plants without a chemical to make sure it doesn't go in. Imo it should do only that, and save the potential issue of emergent chemicals (of which we only have this case rn) for another day.